state/src/applicator/order_book.rs

```diff
@@ -87,7 +87,7 @@ impl StateApplicator {
 
         // Update the order metadata into the `Matching` state
         let wallet = tx
-            .get_wallet_for_order(&order_id)?
+            .get_wallet_id_for_order(&order_id)?
```

Comment:
i changed this method signature and added a separate `get_wallet_for_order`, so these one-liners appear throughout the PR.